scapy/contrib/gtp.py

```diff
@@ -192,14 +192,15 @@ def answers(self, other):
         return (isinstance(other, GTPHeader) and
                 self.version == other.version and
                 self.payload.answers(other.payload))
-
-    @classmethod
-    def dispatch_hook(cls, _pkt=None, *args, **kargs):
-        if _pkt and len(_pkt) >= 1:
-            if (struct.unpack("B", _pkt[0])[0] >> 5) & 0x7 == 2:
-                import gtp_v2
-                return gtp_v2.GTPHeader
-        return GTPHeader
+    

```

Comment:
We can't leave a ^M in the code, it's really ugly and may crash with some python versions...
Have you tried https://github.com/secdev/scapy/pull/531#issuecomment-282588778 ?